### PR TITLE
Added builder class for healthcheck manager Also Issue 13,16

### PR DESCRIPTION
**Pull request by phillbarber**
This pull requests deals with issues 13, (a bit of 14) and 16.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -93,9 +110,14 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.18.1</version>
                 <configuration>
-                    <testSourceDirectory>${basedir}/src/main/java/</testSourceDirectory>
-                    <testClassesDirectory>${project.build.directory}/classes/</testClassesDirectory>
```
Comment (anuragkapur) [inline pom.xml]: This was a side effect from when the library and the sample app were in the same project. The library doesn't need this anymore but I think the sample app will. Anyway, a valid change. Just thought would mention this here for historical documentation.
/cc @phillbarber

File: src/main/java/me/atam/atam4j/AcceptanceTestHealthCheckManager.java [hunk 2]
```diff
@@ -1,44 +1,90 @@
...
 
-    public void initialise() {
-        AcceptanceTestsState acceptanceTestsState = new AcceptanceTestsState();
+    public static class AcceptanceTestsRunnerTaskSchedulerBuilder{
+
+        private AcceptanceTestsState acceptanceTestsState = new AcceptanceTestsState();
+        private Environment environment = null;
+        private Class testClasses[] = null;
+        private long initialDelay = 0;
+        private long period = 1;
```
Comment (anuragkapur) [inline src/main/java/me/atam/atam4j/AcceptanceTestHealthCheckManager.java]: For ease of use, should we configure the default to 10 minutes in seconds? Just thinking of a good default which would make sense if a decent subset of use-cases..?
/cc @phillbarber

Comment (phillbarber) [inline src/main/java/me/atam/atam4j/AcceptanceTestHealthCheckManager.java]: Hi Anurag - I agree - a good library should have good defaults.  10 minutes seems like quite a long period of time to me.  How about 1 minute initial delay, 5 minute period?

Comment (anuragkapur) [inline src/main/java/me/atam/atam4j/AcceptanceTestHealthCheckManager.java]: Sounds good

**Pull request merged**